[2024-07-08T02:26:22.216+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T02:26:22.278+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final.starting scheduled__2024-07-08T02:23:59.582833+00:00 [queued]>
[2024-07-08T02:26:22.292+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final.starting scheduled__2024-07-08T02:23:59.582833+00:00 [queued]>
[2024-07-08T02:26:22.293+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-08T02:26:22.316+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): starting> on 2024-07-08 02:23:59.582833+00:00
[2024-07-08T02:26:22.326+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=62) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T02:26:22.330+0000] {standard_task_runner.py:63} INFO - Started process 67 to run task
[2024-07-08T02:26:22.329+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final', 'starting', 'scheduled__2024-07-08T02:23:59.582833+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpcspjbyqd']
[2024-07-08T02:26:22.332+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask starting
[2024-07-08T02:26:22.426+0000] {task_command.py:426} INFO - Running <TaskInstance: Final.starting scheduled__2024-07-08T02:23:59.582833+00:00 [running]> on host 5c62fe4805cb
[2024-07-08T02:26:22.635+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final' AIRFLOW_CTX_TASK_ID='starting' AIRFLOW_CTX_EXECUTION_DATE='2024-07-08T02:23:59.582833+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-08T02:23:59.582833+00:00'
[2024-07-08T02:26:22.644+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T02:26:22.646+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-08T02:26:22.649+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Chuẩn bị cào dữ liệu"']
[2024-07-08T02:26:22.678+0000] {subprocess.py:86} INFO - Output:
[2024-07-08T02:26:22.689+0000] {subprocess.py:93} INFO - Chuẩn bị cào dữ liệu
[2024-07-08T02:26:22.690+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-08T02:26:22.692+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T02:26:22.758+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Final, task_id=starting, run_id=scheduled__2024-07-08T02:23:59.582833+00:00, execution_date=20240708T022359, start_date=20240708T022622, end_date=20240708T022622
[2024-07-08T02:26:22.828+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T02:26:22.885+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T02:26:22.889+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
